lib/internal/Magento/Framework/Model/ResourceModel/AbstractResource.php

```diff
@@ -189,6 +189,17 @@ protected function _prepareDataForTable(DataObject $object, $table)
         return $data;
     }
 
+    /**
+     * @param string $column
+     * @param string $table
+     * @return array
+     */
+    protected function _getColumnDefinition($column, $table)
```

Comment:
Unfortunately, we are not allowed to add new protected methods to the @api classes according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)